examples/fedopt_example/client.py

```diff
@@ -68,9 +68,12 @@ def setup_client(self, config: Config) -> None:
                 metric.setup(self.label_encoder)
         super().setup_client(config)
 
-    def predict(self, input: torch.Tensor) -> Tuple[Dict[str, torch.Tensor], Dict[str, torch.Tensor]]:
+    def predict(
+        self,
+        input: TorchInputType,
```

Comment:
If we're going to be moving to this type alias, I'd suggest we have method documentation discussing the type and functionality. That is, this is just a union of `torch.Tensor `and `Dict[str, torch.Tensor]`, since these are supposed to be examples for people to follow and maybe they'll have less intimate knowledge of the library itself.

Reply:
I think the type alias is worthwhile though!

Reply:
Good suggestion!